lec15-git-projects.Rmd

```diff
@@ -29,121 +29,133 @@ author: Ahmed Hasan
-reproducibility, it has also become increasingly common for scientists to upload 
-scripts and related files to GitHub for others to use.
-
-In this lesson, we'll be covering some basic Git commands, before
-moving on to using GitHub for the upcoming group projects.
-All members of the group will be given admin access to a pre-made repository. 
+Git is a version control system that tracks changes in files. Although it is
+primarily used for software development, Git can be used to track changes in
+any files, and is an indispensable tool for collaborative projects. Using Git,
+we effectively create different versions of our files and track who has made
+what changes. The complete history of the changes for a particular project and
+their metadata make up a repository. 
+
+To sync the repository across different computers and to collaborate with
+others, Git is often used via [GitHub](http://www.github.com), a web-based
+service that hosts Git repositories. In the spirit of 'working open' and
+ensuring scientific reproducibility, it has also become increasingly common for
+scientists to upload scripts and related files to GitHub for others to use.
+
+In this lesson, we'll be covering some basic Git commands, before moving on to
+using GitHub for the upcoming group projects.  All members of the group will be
+given admin access to a pre-made repository. 
 
 ### Command line Git
 
 #### Setup and Installation
 
-Git is primarily used on the command line. The implementation of the
-command line that we'll be using is known as a 'bash interpreter', or
-'shell'. While bash interpreters are natively available on Mac and Linux 
-operating systems, Windows users may have to externally install a bash shell.
-
-To install Git on a **Mac**, you will have to install [Xcode](https://itunes.apple.com/ca/app/xcode/id497799835?mt=12),
-which can be downloaded from the Mac App store. Be warned: Xcode is a very 
-large download (approximately 6 GB)! Install Xcode and Git should be
-ready for use. (_Note: most students will have already installed Xcode
-already for some R packages we used earlier in the course. If you're not
-sure whether this is the case, run the `git --version` command described
-below_)
-
-To install Git (and Bash) on **Windows**, download [Git for Windows](https://gitforwindows.org/) from its [website](https://github.com/git-for-windows/git/releases/tag/v2.19.1.windows.1). 
+Git is primarily used on the command line. The implementation of the command
+line that we'll be using is known as a 'bash interpreter', or 'shell'. While
+bash interpreters are natively available on Mac and Linux operating systems,
+Windows users may have to externally install a bash shell.
+
+To install Git on a **Mac**, you will have to install
+[XCode](https://itunes.apple.com/ca/app/xcode/id497799835?mt=12), which can be
```

Comment:
Wiki seems to have renamed it XCode->Xcode.

Reply:
Dammit, Apple...